package.json

```diff
@@ -41,6 +44,9 @@
     "redux": "^4.0.1",
     "redux-thunk": "^2.3.0"
   },
+  "resolutions": {
```

Comment:
We already have babel/core under devDependencies.
https://github.com/stavalfi/workflow/blob/c5ecb730a7c506769c71e02b19e53ff531d1dc1f/package.json#L19

Reply:
https://github.com/facebook/jest/issues/6913

Reply:
then remove "@babel/core": "^7.1.5",  and install "babel-core": "7.0.0-bridge.0" instead under devDependencies ? or is it a problem also? 

Choose any solution you want but try to remove one of them.